wiki/History_of_osu!/2011/en.md

```diff
@@ -1,9 +1,54 @@
+
+The first edition of [osu!monthly](/wiki/Community/osu!monthly) was launched by ::{ flag=US }:: [jjrocks](https://osu.ppy.sh/users/30985) and ::{ flag=GB }:: [Corin](https://osu.ppy.sh/users/639270), presenting a variety of content ranging from user statistics to promotions in the [Beatmap Appreciation Team (BAT)](/wiki/People/Beatmap_Appreciation_Team) and the [Mapping Assistance Team (MAT)](/wiki/People/Mapping_Assistance_Team).[^monthly-1]
+
+## August
+
+[osu!stream](/wiki/osu!stream) was published, making osu! available on [iOS](https://en.wikipedia.org/wiki/IOS) with unique features. Compared to the desktop version, osu!stream offered new gameplay mechanics and original beatmaps.[^monthly-5]
+
+[LuigiHann](https://osu.ppy.sh/users/1079) organised the first official storyboarding contest, where users were asked to create [storyboards](/wiki/Storyboard) for the three finalists of the [Official Beatmapping Contest 3](https://osu.ppy.sh/community/forums/topics/47404?n=1).[^storyboarding]
+
+## September
+
+An unofficial version of osu! was released by [Pesets](https://osu.ppy.sh/users/%40Pesets) for the [Android](https://en.wikipedia.org/wiki/Android_(operating_system)) platform.[^droid] A dedicated [subforum](http://osu.ppy.sh/forum/viewforum.php?f=85) for this project is now archived.
+
+## October
+
+The count of registered users surpassed 1,000,000.[^monthly-6]
 
 ## December
 
-The osu! wiki was announced, allowing users to add all sorts of information about everything osu! related.
+The osu! wiki was announced, allowing users to add all sorts of information about everything osu! related.[^wiki1][^wiki2]
+
+## References
+
+[^web-update]: [Forum thread by peppy (2011-02-16) "osu! web updates"](https://osu.ppy.sh/community/forums/topics/46571?n=1)
+
+[^monthly-1]: [osu!monthly by jjrocks (2011-03-05) "The osu!monthly 1st edition"](https://osumonthly.com/pdf/osu!monthly-1.pdf)
+[^test-builds]: [Forum thread by peppy (2011-03-09) "Public Test Build"](https://osu.ppy.sh/community/forums/topics/48054?n=1)
+[^bancho]: [Forum thread by peppy (2011-03-20) "The state of Bancho"](https://osu.ppy.sh/community/forums/topics/48908?n=1)
+[^stable-b1800]: [Forum thread by peppy (2011-03-28) "osu! Public Release b1800"](https://osu.ppy.sh/community/forums/topics/49438?n=1)
+
+[^storyboarding]: [Forum thread by LuigiHann (2011-06-04) "First Οfficial Storyboarding Contest! Deadline August 31"](https://osu.ppy.sh/community/forums/topics/54468?n=1)
+[^monthly-5]: [osu!monthly by jjrocks (2011-08-15) "osu!monthly August 2011 Edition"](https://osumonthly.com/pdf/osu!monthly-5.pdf)
+
+[^droid]: [Forum thread by peppy (2011-09-19) "osu!droid released!"](https://osu.ppy.sh/community/forums/topics/62561?n=1)
 
-Links:
+[^monthly-6]: [osu!monthly by jjrocks (2011-10-27) "osu!monthly October 2011 Edition"](https://osumonthly.com/pdf/osu!monthly-6.pdf)
```

Comment:
this citation format will undoubtedly see more edge cases but i guess we'll make it up as we go (unless we end up switching to an existing one)

very open to suggestions. this one has many authors and the correct title isn't entirely clear

```suggestion
[^monthly-6]: [Magazine by jjrocks (2011-10-27) "osu!monthly October 2011 Edition"](https://osumonthly.com/pdf/osu!monthly-6.pdf)
```